modules/widget-html-render.twig

```diff
@@ -50,11 +50,17 @@
             }
         </style>
         {% if style|length > 0 %}
-        <style type="text/css">
             {% for item in style %}
-            {{ item|raw }}
+                {% if item.content %}
+                    <style type="text/css" data-style-scope="{{item.type}}_{{item.dataType}}__{{item.templateId}}">
+                        {{ item.content|raw }}
+                    </style>
+                {% else %}
```

Comment:
This should never be the case because the WidgetHtmlRenderer has been changed at the same time.